lib/command.js

```diff
@@ -1559,6 +1590,26 @@ Expecting one of '${allowedValues.join("', '")}'`);
     this.error(message, { code: 'commander.missingMandatoryOptionValue' });
   }
 
+  /**
+   * `Option` conflicts with another option.
+   *
+   * @param {Option} option
+   * @api private
+   */
+  exclusiveOptionPresent(option, exclusive) {
```

Comment:
This method can start with an underscore to show it is private. (The older routines don't have an underscore, but you can see the convention with more recent `_excessArguments`.)

Reply:
Great catch! Done in https://github.com/tj/commander.js/pull/1678/commits/adabd8750161322c38101cbfcfaf2ef3d8aa5026